designs/node-repair.md

```diff
@@ -0,0 +1,93 @@
+#  Karpenter Node Auto Repair 
```

Comment:
```suggestion
# Node Auto Repair 
```

The fact that this is being proposed in the Karpenter project implies that this is a Karpenter design so I'd drop that bit